extension-base/src/main/java/com/azure/autorest/extension/base/plugin/NewPlugin.java

```diff
@@ -192,8 +194,10 @@ public NewPlugin(Connection connection, String plugin, String sessionId)
                 .withIsGetterVisibility(JsonAutoDetect.Visibility.NONE));
         Representer representer = new Representer();
         representer.getPropertyUtils().setSkipMissingProperties(true);
-        Constructor constructor = new CodeModelCustomConstructor();
-        yamlMapper = new Yaml(constructor, representer);
+        LoaderOptions loaderOptions = new LoaderOptions();
+        loaderOptions.setMaxAliasesForCollections(Integer.MAX_VALUE);
```

Comment:
Since 1.26 the default max is 50 (my guess is to fight DDOS). However this small value fails for even simple m4 output.